Makefile

```diff
@@ -112,10 +115,26 @@ env-file:
 	rm $(ENV_FILE)_tmp
 
 .PHONY: install-prereq
-install-prereq: deploy-capp
+install-prereq: deploy-capp install-cluster-proxy-role install-cluster-gateway-role
 
 .PHONY: uninstall-prereq
-uninstall-prereq: undeploy-capp
+uninstall-prereq: undeploy-capp uninstall-cluster-proxy-role uninstall-cluster-gateway-role
+
+.PHONY: install-cluster-proxy-role
+install-cluster-proxy-role:
+	kubectl apply -f $(CLUSTER_PROXY_RBAC)
+
+.PHONY: uninstall-cluster-proxy-role
+uninstall-cluster-proxy-role:
+	kubectl delete -f $(CLUSTER_PROXY_RBAC)
+
+.PHONY: install-cluster-gateway-role
+install-cluster-gateway-role:
+	kubectl apply -f $(CLUSTER_GATEWAY_RBAC)
+
+.PHONY: uninstall-cluster-gateway-role
+uninstall-cluster-gateway-role:
+	kubectl delete -f $(CLUSTER_GATEWAY_RBAC)
```

Comment:
Do we need both?

Reply:
Yes, I added it after seeing this failure: [link](https://github.com/dana-team/platform-backend/actions/runs/10831513878/job/30058147698#step:15:894).